benchmark_scripts/bm_ind.slurm

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#SBATCH --time=24:00:00
+#SBATCH --nodes=1
+#SBATCH --ntasks=16
+
+DATADIR=$(pwd)
+
+
+number=$1
+file=$2
+
+mpirun -np $number quick.MPI $DATADIR/benchmark/$number$file.in
+output=$(cat $DATADIR/benchmark/$number$file.out | grep "cpu time")
```

Comment:
It may be better to use the slurm batch jobs only to run the calculation. And use a separate script to collect timing statistics. However, it would be good to collect timings during the batch job to double check against internal timers. This could be done like
```
tic=`date +%s`
mpirun -np ...
toc=`date +%s`
let "elapsed = $toc - $tic"
echo ">>> elapsed time = $elapsed s"
```
The echo command will output to stdout which gets by default redirected to ${SLURM_JOB_ID}.out. This will redirect stdout/stderr to ${SLURM_JOB_NAME}.out/err
```
#SBATCH --output=%x.out
#SBATCH --error=%x.err
```